src/main/target/KROOZX/config.c

```diff
@@ -36,6 +36,8 @@ void targetConfiguration(void)
 	voltageSensorADCConfigMutable(VOLTAGE_SENSOR_ADC_VBAT)->vbatscale = VBAT_SCALE;
     barometerConfigMutable()->baro_hardware = 0;
     compassConfigMutable()->mag_hardware = 0;
-    osdConfigMutable()->item_pos[OSD_MAIN_BATT_VOLTAGE] = OSD_POS(12, 1) | VISIBLE_FLAG;
+    osdConfigMutable()->item_pos[OSD_MAIN_BATT_VOLTAGE][0] = 12;
+    osdConfigMutable()->item_pos[OSD_MAIN_BATT_VOLTAGE][1] = 1;
```

Comment:
0 and 1 are magic numbers, replace with defines for X and Y or ROW/COLUMN as appropriate.